packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -273,6 +284,17 @@ final LogicalKeySet goToLineNumberKeySet = LogicalKeySet(
   LogicalKeyboardKey.keyG,
 );
 
+final LogicalKeySet searchInFileKeySet = LogicalKeySet(
```

Comment:
We don't have to do in this PR but it's probably worth breaking all of the keybindings out into a single file so that they are easy to find.

Reply:
added a todo for this